### DISCUSSION TITLE
Password

### DISCUSSION THREAD

**Discussion by Ossy63** [iotaledger/firefly#7671] [Help & Support]
Shimmer/Firefly don't know my password. Always came: 

"Stronghold migration error; failed to decrypt snapshot; incorrect password or corrupt data"

Please help, what can I do?

Greetings, Oswald

**Comment by begonaalvarezd** [comment]
That error message suggests that indeed you are not using the correct password. Please make sure you are using the correct password, and if you dont remember it, you can always restore your profile from a stronghold backup for which you do have the password, or using your 24 words mnemonic

**Comment by Ossy63** [comment]
Thx for your answer! 
I have installed Firefly-Shimmer whitout any password.
I didn't assign a password and I didn't get any codes. 
Are they the same 24 words from Firefly-IOTA? Is the stronghold password from Firefly_IOTA not the same?

**Reply by begonaalvarezd** [reply]
When you create a software profile in firefly it always asks to set a password. Firefly IOTA & Firefly shimmer are different apps, if you restored in firefly shimmer from a firefly iota stronghold, it will use the same password you had in the imported iota stronghold, unless you changed it. If you created brand new profiles, each will have its own password, unless you used the same in both

**Reply by begonaalvarezd** [reply]
The 24 words are not the same as the passowrd, your 24 words are your seed(never share this), the password is the password to encrypt the stronghold file (is like a cold wallet, but software instead of hardware) which is used in the software profiles
